composer.json

```diff
@@ -15,6 +15,7 @@
         "php": "^7.1",
         "ext-json": "*",
         "ext-mbstring": "*",
+        "jean85/pretty-package-versions": "^1.2",
```

Comment:
This is now a root requirement. It has the side-effect of making the `ModulesIntegration` work out-of-the-box, and it could be also leveraged by other integrations to set the version in a faster and more reliable way too.

Reply:
I don't have anything against your package but I really don't want to add another required dep for the SDK to work.
We already have this `php composer.phar require sentry/sentry php-http/curl-client guzzlehttp/psr7`
Ideally we would only have `sentry/sentry`.

Reply:
This isn't to be added anywhere while installing, is required so it would be implied in the `composer require sentry/sentry`. The other package is needed to specify which HTTP client to use, since it's up to the user to chose that.

Reply:
I agree with @HazAT that we should avoid introducing other dependencies not strictly needed. In this case I see the utility of the package but I believe that it's not essential and that remembering to bump a constant before release would not be such a drama

Reply:
So tbh if we don't need to tell people that they have to add the package to their `composer` call like: 
`php composer.phar require sentry/sentry php-http/curl-client guzzlehttp/psr7 jean85/pretty-package-versions` then this can stay. If they have to add it I vote for removing it.

Reply:
No, it's not needed to be added manually, it's installed automatically as as sub-dependency of `sentry/sentry`. On the contrary, installation would have been needed manually if you wanted to use the `ModulesIntegration`; not anymore with this change.